app/main.py

```diff
@@ -2,5 +2,17 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    funcs = {}
```

Comment:
`funcs` variable name is not representative enough, replace it with a name that would be more applicable, e.g. `cached_results`.